### PR TITLE
Create 2021-February-1.md

### PR DESCRIPTION
**Pull request by kagbakpem**
Android Release notes for ACS Calling (Jan release)

### CODE REVIEW

File: releasenotes/2021-February-1.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    callAgentOptions.setDisplayName("My Custom Display name");
+    CommunicationTokenCredential tokenCredential = new CommunicationTokenCredential("<USER ACCESS TOKEN>");
+    CallAgent callAgent = callClient.createCallAgent(tokenCredential, callAgentOptions);
+
+    android.content.context appContext = this.getApplicationContext(); // From within an activity or fragment
+    string threadId = <TEAMS THREAD ID>;
+    java.util.UUID organizerId = java.util.UUID.fromString(<TEAMS ORGANIZER ID>);
+    java.util.UUID tenantId = java.util.UUID.fromString(<TEAMS TENANT ID>);
+    string messageId = <TEAMS MESSAGFE ID>;
+    public TeamsMeetingCoordinatesLocator(String threadId, java.util.UUID organizerId, java.util.UUID tenantId, String messageId)
```
Comment (ashishvirani7) [inline releasenotes/2021-February-1.md:49]: Is this code intended?

File: releasenotes/2021-February-1.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+## Breaking API changes
+`CallAgent.join` API has been changed to support Teams interoperability
+`join(android.content.Context context, GroupCallContext groupCallContext, JoinCallOptions joinCallOptions)`
+is now:
+`join(android.content.Context context, AbstractJoinMeetingLocator meetingLocator, JoinCallOptions joinCallOptions)`
+
+With new types `GroupCallLocator`, `TeamsMeetingCoordinatesLocator`, `TeamsMeetingLinkLocator` being subclasses of `AbstractJoinMeetingLocator` that can be used for the various scenarios.
+
+### Usage example with `GroupCallLocator`
```
Comment (ashishvirani7) [inline releasenotes/2021-February-1.md:83]: Why adding this block again?

File: releasenotes/2021-February-1.md (hunk 1 above)
Comment (kagbakpem) [reply releasenotes/2021-February-1.md:52]: which line?

**Review by ashishvirani7 — APPROVED** [review]

File: releasenotes/2021-February-1.md (hunk 2 above)
Comment (kagbakpem) [reply releasenotes/2021-February-1.md:83]: It's under a different heading `Breaking Changes` vs `New Features`. I could instead to the earlier reference.

**Pull request merged**